libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -372,7 +372,7 @@ - (void)kickOffIDPInitiatedLoginFlowForSP:(SFSDKSPConfig *)config statusUpdate:(
         return;
     }
 
-    NSString *challengeString = [[[[codeVerifier msdkBase64UrlString] dataUsingEncoding:NSUTF8StringEncoding] msdkSha256Data] msdkBase64UrlString];
+    NSString *challengeString = [[[codeVerifier msdkBase64UrlString] sha256] msdkBase64UrlString];
```

Comment:
this one generates the same string, just realized we had a more convenient method